_ml-commons-plugin/algorithms.md

```diff
@@ -629,4 +629,75 @@ POST _plugins/_ml/_predict/logistic_regression/SsfQaoIBEoC4g4joZiyD
 
 ### Limitations
 
-Convergence metrics are not built into Tribuo's trainers. Therefore, ML Commons cannot indicate the convergence status through the ML Commons API.
\ No newline at end of file
+Convergence metrics are not built into Tribuo's trainers. Therefore, ML Commons cannot indicate the convergence status through the ML Commons API.
+
+## Metrics correlations
+
+The metrics correlations feature is an experimental feature released in OpenSearch 2.7. It can't be used in a production environment. To leave feedback on improving the feature, leave an issue in the [ML Commons repository](https://github.com/opensearch-project/ml-commons).
+{: .warning }
+
+The metrics correlation algorithm finds events in a set of metrics data. The algorithm defines events as a window in time where multiple metrics simultaneously display anomalous behavior. When given a time span and set of metrics, The algorithm correlates the number of events that occurred, when the event occurred, and determines which metrics were involved in each event.
+
+To enable the metrics correlation algorithm, update the following cluster setting:
+
+```
+PUT /_cluster/settings
+{
+  "persistent" : {
+    "plugins.ml_commons.enable_inhouse_python_model": true
+  }
+}
+```
+
+### Parameters
+
+To use the metrics correlations algorthim, include the following parameters.
+
+| Parameter | Type | Description | Default Value |
+|---|---|---|---|
+metrics | Integer | A list of metrics within the time series that can be correlated to anomalous behavior | N/A
```

Comment:
input is a list of metrics. Every metrics is an array of numbers/float values. 

A simplest example can be: `{"metrics": [[1.0, 2.0, 3.0], [1.0, 2.0, 3.0]]}`, here 2 metrics are provided and each metrics has 3 datapoints.